lib/model/query/submissions.js

```diff
@@ -134,20 +153,34 @@ order by path asc, value asc`)
 ////////////////////////////////////////////////////////////////////////////////
 // SUBMISSION GETTERS
 
-const _get = extender(Submission)(Actor.into('submitter'))((fields, extend, options, projectId, xmlFormId, draft) => sql`
-select ${fields} from submissions
-join forms on forms."xmlFormId"=${xmlFormId} and forms.id=submissions."formId"
-join projects on projects.id=${projectId} and projects.id=forms."projectId"
-${extend|| sql`left outer join actors on actors.id=submissions."submitterId"`}
-where ${equals(options.condition)} and draft=${draft} and submissions."deletedAt" is null
-order by submissions."createdAt" desc, submissions.id desc
-${page(options)}`);
+// Helper function to assign submission.currentVersionSubmitter to submission.currentVersion.submitter
+// Current there is no way to create such complex object using `extender` and `unjoiner` framework functions
+const assignCurrentVersionSubmitter = (x) => x.withAux('currentVersion', x.aux.currentVersion.withAux('submitter', x.aux.currentVersionSubmitter));
+
+const _get = extender(Submission, Submission.Def.into('currentVersion'))(Actor.into('submitter'), Actor.alias('current_version_actors', 'currentVersionSubmitter'))((fields, extend, options, projectId, xmlFormId, draft) => sql`
```

Comment:
I might be wrong here, do you know if there is a way to use `extender` to set `currentVersion.submitter`?

Reply:
I'm not sure actually! I can't think of a similar example of nested extended metadata, so maybe not.